flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/api/listener/FlinkResourceListener.java

```diff
@@ -45,8 +44,6 @@ public interface FlinkResourceListener extends Plugin {
     interface ResourceContext<R extends AbstractFlinkResource<?, ?>> {
         R getFlinkResource();
 
-        KubernetesClient getKubernetesClient();
```

Comment:
I think we should not touch this here, I think the FlinkResourceListener interfaces should be reworked but probably not as part of this ticket.

Reply:
Thanks for the comment, have addressed it in latest commit